README.md

```diff
@@ -88,6 +89,22 @@ lru_redux thread safe   2.190000   0.010000   2.200000 (  2.185512)
 
 ```
 
+#### TTL
+```
+$ ruby ./bench/bench_ttl.rb
+Rehearsal -----------------------------------------------------------------------
+FastCache                             6.240000   0.070000   6.310000 (  6.302569)
+LruRedux::TTL::Cache                  4.700000   0.010000   4.710000 (  4.712858)
```

Comment:
I will add an alternatives section.  Another thing to note about FastCache is that it has a bug on how TTL ordering is stored and LruRedux benchmark is the worst case scenario.  I am not sure if it even affects the bench time, and if it does, whether it would make it faster or slower.